CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ Changelog for NeoFS Node
 - Expose health status of inner ring via Prometheus (#2934)
 
 ### Fixed
+- `defer` func is not skipped in cobra-based programs (#2942)
```

Comment:
Not sure it makes sense for outside reader, but don't have immediate suggestions either.